{
  "Faults": [
    {
      "latitude": 51.633360827455974,
      "type": "PSI",
      "reference": "213829",
      "etaUpdate": true,
      "affectedCustomerCount": 5,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "f66b546d-fa15-4747-b2e2-84c8ca92443b",
      "minLongitude": -1.8342,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.8077456535964522,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-03-06T08:00:00.000+0000",
      "estimatedRestoration": "2023-03-09T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '213829'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/213829",
      "maxLongitude": -1.7716,
      "minLatitude": 51.6139,
      "name": "SN26 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN26 8DD",
        "SN6 6JX"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.8342,
              51.6603
            ],
            [
              -1.7716,
              51.6603
            ],
            [
              -1.7716,
              51.6139
            ],
            [
              -1.8342,
              51.6139
            ],
            [
              -1.8342,
              51.6603
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.6603,
      "etrUpdate": true
    },
    {
      "latitude": 51.56208195216424,
      "type": "LV",
      "reference": "KQ2873",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-03-06T10:20:00.000+0000",
      "affectedCustomerCount": 17,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "53e97a7a-8c88-4528-9f9f-b079a608f414",
      "minLongitude": -1.7592,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7567622821779547,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-03-06T09:37:00.000+0000",
      "estimatedRestoration": "2023-03-06T22:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KQ2873'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KQ2873",
      "maxLongitude": -1.7554,
      "minLatitude": 51.5608,
      "name": "SN3 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN3 3BY",
        "SN3 3DP",
        "SN3 3DR",
        "SN3 3DS",
        "SN3 3DT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.7592,
              51.5646
            ],
            [
              -1.7554,
              51.5646
            ],
            [
              -1.7554,
              51.5608
            ],
            [
              -1.7592,
              51.5608
            ],
            [
              -1.7592,
              51.5646
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-03-06T10:20:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5646,
      "etrUpdate": true
    },
    {
      "latitude": 51.76004167258937,
      "type": "LV",
      "reference": "KQ3106",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-03-06T13:00:00.000+0000",
      "affectedCustomerCount": 14,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "34db4a2c-d36c-4a48-b14e-c9fed3290056",
      "minLongitude": -1.2166,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.212932190644,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-03-06T11:42:00.000+0000",
      "estimatedRestoration": "2023-03-06T21:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KQ3106'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KQ3106",
      "maxLongitude": -1.2109,
      "minLatitude": 51.7588,
      "name": "OX3 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "OX3 7RD",
        "OX3 7RF",
        "OX3 9AA",
        "OX3 9AE",
        "OX3 9AH",
        "OX3 9AY",
        "OX3 9FA"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.2166,
              51.7621
            ],
            [
              -1.2109,
              51.7621
            ],
            [
              -1.2109,
              51.7588
            ],
            [
              -1.2166,
              51.7588
            ],
            [
              -1.2166,
              51.7621
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-03-06T12:40:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.7621,
      "etrUpdate": true
    },
    {
      "latitude": 51.84407875534984,
      "type": "HV",
      "reference": "KQ3289",
      "etaUpdate": true,
      "affectedCustomerCount": 12,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "a5d00b25-60c4-456b-89f3-034f6f406174",
      "minLongitude": -1.5607,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.4232703888843143,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-03-06T13:45:00.000+0000",
      "estimatedRestoration": "2023-03-06T21:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KQ3289'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KQ3289",
      "maxLongitude": -1.3232,
      "minLatitude": 51.7224,
      "name": "OX20 & OX28 Areas",
      "otherUpdate": true,
      "affectedAreas": [
        "OX20 1HD",
        "OX20 1QA",
        "OX28 1BH",
        "OX28 1BL",
        "OX28 1BP",
        "OX28 3JJ",
        "OX29 6QA",
        "OX29 6QD",
        "OX29 6QF",
        "OX29 7QE",
        "OX29 7RT",
        "OX29 8BF",
        "OX29 8BT",
        "OX29 8BZ",
        "OX29 8DN",
        "OX29 8DR",
        "OX29 8DS",
        "OX29 8DT",
        "OX29 8DU",
        "OX29 8DW",
        "OX29 8DX",
        "OX29 8DY",
        "OX29 8DZ",
        "OX29 8EA",
        "OX29 8EB",
        "OX29 8ED",
        "OX29 8EE",
        "OX29 8EF",
        "OX29 8EG",
        "OX29 8EH",
        "OX29 8EJ",
        "OX29 8EL",
        "OX29 8EN",
        "OX29 8EP",
        "OX29 8EQ",
        "OX29 8ER",
        "OX29 8ES",
        "OX29 8ET",
        "OX29 8EU",
        "OX29 8EW",
        "OX29 8EX",
        "OX29 8EZ",
        "OX29 8FA",
        "OX29 8FB",
        "OX29 8FF",
        "OX29 8FG",
        "OX29 8FH",
        "OX29 8FJ",
        "OX29 8GE",
        "OX29 8GG",
        "OX29 8HA",
        "OX29 8HB",
        "OX29 8LA",
        "OX29 8LS",
        "OX29 8NA",
        "OX29 8NB",
        "OX29 8ND",
        "OX29 8NE",
        "OX29 8NF",
        "OX29 8NG",
        "OX29 8NH",
        "OX29 8NJ",
        "OX29 8NL",
        "OX29 8NN",
        "OX29 8NP",
        "OX29 8NQ",
        "OX29 8NS",
        "OX29 8NT",
        "OX29 8NU",
        "OX29 8NW",
        "OX29 8NX",
        "OX29 8NY",
        "OX29 8NZ",
        "OX29 8PA",
        "OX29 8PB",
        "OX29 8PD",
        "OX29 8PE",
        "OX29 8PF",
        "OX29 8PG",
        "OX29 8PH",
        "OX29 8PJ",
        "OX29 8PL",
        "OX29 8PN",
        "OX29 8PP",
        "OX29 8PR",
        "OX29 8PS",
        "OX29 8PT",
        "OX29 8PU",
        "OX29 8PW",
        "OX29 8PX",
        "OX29 8PY",
        "OX29 8PZ",
        "OX29 8QA",
        "OX29 8QB",
        "OX29 8QD",
        "OX29 8QE",
        "OX29 8QF",
        "OX29 8QG",
        "OX29 8QH",
        "OX29 8QJ",
        "OX29 8QN",
        "OX29 8QQ",
        "OX29 8QT",
        "OX29 8QU",
        "OX29 8QW",
        "OX29 8QX",
        "OX29 8QY",
        "OX29 8QZ",
        "OX29 8RH",
        "OX29 8ST",
        "OX29 9PY",
        "OX7 4EU",
        "OX7 4EX",
        "OX8"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.5607,
              51.88
            ],
            [
              -1.3232,
              51.88
            ],
            [
              -1.3232,
              51.7224
            ],
            [
              -1.5607,
              51.7224
            ],
            [
              -1.5607,
              51.88
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.88,
      "etrUpdate": true
    },
    {
      "latitude": 51.351421380544835,
      "type": "LV",
      "reference": "KQ3538",
      "etaUpdate": true,
      "affectedCustomerCount": 17,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "a1754f81-1c64-4f8b-b664-f1da5b1f0b60",
      "minLongitude": -0.5987,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.5958182284962048,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-03-06T17:12:00.000+0000",
      "estimatedRestoration": "2023-03-06T21:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KQ3538'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KQ3538",
      "maxLongitude": -0.575,
      "minLatitude": 51.345,
      "name": "GU24 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GU24 8PJ",
        "GU24 8PL",
        "GU24 8PN",
        "GU24 8PP",
        "GU24 8PT",
        "GU24 8PU",
        "GU24 8PW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.5987,
              51.3536
            ],
            [
              -0.575,
              51.3536
            ],
            [
              -0.575,
              51.345
            ],
            [
              -0.5987,
              51.345
            ],
            [
              -0.5987,
              51.3536
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-03-06T17:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.3536,
      "etrUpdate": true
    },
    {
      "latitude": 50.84426101261292,
      "type": "LV",
      "reference": "KQ3589",
      "etaUpdate": true,
      "affectedCustomerCount": 11,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "78a4abed-93cd-431f-809a-2704414e1dda",
      "minLongitude": -1.067,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.0648960075566718,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-03-06T19:25:00.000+0000",
      "estimatedRestoration": "2023-03-07T01:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KQ3589'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KQ3589",
      "maxLongitude": -1.0631,
      "minLatitude": 50.8436,
      "name": "PO6 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO6 2QB",
        "PO6 2QG",
        "PO6 2QR"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.067,
              50.8447
            ],
            [
              -1.0631,
              50.8447
            ],
            [
              -1.0631,
              50.8436
            ],
            [
              -1.067,
              50.8436
            ],
            [
              -1.067,
              50.8447
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-03-06T20:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.8447,
      "etrUpdate": true
    },
    {
      "latitude": 51.41021193454624,
      "type": "LV",
      "reference": "KQ3588",
      "etaUpdate": true,
      "affectedCustomerCount": 2,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "ad23656f-a690-4d87-bd44-e10f11789491",
      "minLongitude": -2.2014,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.1971014246194387,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-03-06T19:23:00.000+0000",
      "estimatedRestoration": "2023-03-07T01:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KQ3588'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KQ3588",
      "maxLongitude": -2.1906,
      "minLatitude": 51.403,
      "name": "SN13 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN13 9TD",
        "SN13 9TE"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.2014,
              51.4115
            ],
            [
              -2.1906,
              51.4115
            ],
            [
              -2.1906,
              51.403
            ],
            [
              -2.2014,
              51.403
            ],
            [
              -2.2014,
              51.4115
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-03-06T20:10:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4115,
      "etrUpdate": true
    },
    {
      "latitude": 56.57716842356003,
      "type": "HV",
      "reference": "KQ3179",
      "etaUpdate": true,
      "affectedCustomerCount": 10,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "4790ebdb-8e8a-4a28-8497-20d0c4c72804",
      "minLongitude": -5.3544,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -5.22183470288556,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-03-06T12:40:00.000+0000",
      "estimatedRestoration": "2023-03-06T20:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KQ3179'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KQ3179",
      "maxLongitude": -5.1033,
      "minLatitude": 56.5167,
      "name": "PA38 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PA38 4BJ",
        "PA38 4BQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -5.3544,
              56.6507
            ],
            [
              -5.1033,
              56.6507
            ],
            [
              -5.1033,
              56.5167
            ],
            [
              -5.3544,
              56.5167
            ],
            [
              -5.3544,
              56.6507
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-03-06T13:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 56.6507,
      "etrUpdate": true
    },
    {
      "latitude": 58.33920194041272,
      "type": "HV",
      "reference": "KQ3555",
      "etaUpdate": true,
      "affectedCustomerCount": 34,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "8c2fb356-0d38-41fe-9871-109b1fc98df9",
      "minLongitude": -6.673,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -6.649777628996344,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-03-06T17:48:00.000+0000",
      "estimatedRestoration": "2023-03-06T20:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KQ3555'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KQ3555",
      "maxLongitude": -6.6072,
      "minLatitude": 58.2982,
      "name": "HS2 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "HS2 9DD",
        "HS2 9DH"
      ],
      "location": {
        "coordinates": [
          [
            [
              -6.673,
              58.3543
            ],
            [
              -6.6072,
              58.3543
            ],
            [
              -6.6072,
              58.2982
            ],
            [
              -6.673,
              58.2982
            ],
            [
              -6.673,
              58.3543
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 58.3543,
      "etrUpdate": true
    }
  ]
}
